charmcraft.yaml

```diff
@@ -35,27 +35,44 @@ requires:
     optional: false
     limit: 1
 
+actions:
+  blacklist-and-delete-keys:
+    description: Blacklist and delete keys from the keyserver database.
+    properties:
+      fingerprints:
+        type: string
+        description: Comma-separated list of full fingerprints to ignore (minus the leading 0x).
+      comment:
+        type: string
+        description: Any comment to pass along with the action.
+    required: 
+      - fingerprints
+      - comment
+  rebuild-prefix-tree:
+    description: |
+      The prefix tree is used to manage the reconciliation process and 
+      must always be in sync with the database. If you suspect it has become corrupted or out 
```

Comment:
s/has become corrupted/ is corrupted. When writing technical documentation, you want to keep your grammar as simple as possible to that it is accessible to a wider audience :)